caffe2/__init__.py

```diff
@@ -0,0 +1,6 @@
+import warnings
+from torch.onnx import _CAFFE2_ATEN_FALLBACK
+
+if not _CAFFE2_ATEN_FALLBACK:
+    warnings.warn("Caffe2 support is not fully enabled in this PyTorch build. "
```

Comment:
```suggestion
    warnings.warn("Caffe2 was not built with this PyTorch build. "
```
Very likely i'm missing something here. Does `fully enabled` means Caffe2 is still partially enabled?

Reply:
It is always possible to do `import caffe2` when pytorch is installed, even if pytorch was not compiled with caffe2 support.

Bu fully enabled I mean libcaffe2 and other libraries are not enabled (compiled)